File: app/src/main/java/com/vcheck/demo/dev/presentation/screens/LivenessInstructionsFragment.java
package com.vcheck.demo.dev.presentation.screens;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.navigation.NavController;
import androidx.navigation.Navigation;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.vcheck.demo.dev.R;


public class LivenessInstructionsFragment extends Fragment {

    public LivenessInstructionsFragment() {

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.liveness_instructions_fragment, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        NavController navController = Navigation.findNavController(view);

        ImageView checkButton = view.findViewById(R.id.check_button);
        checkButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                navController.navigate(R.id.action_mainFragment_to_faceCheckFragment);
            }
        });
    }
}

File: app/src/main/java/com/vcheck/demo/dev/presentation/liveness/ui/LivenessInstructionsFragment.kt
package com.vcheck.demo.dev.presentation.liveness.ui

import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.findNavController
import com.vcheck.demo.dev.R
import com.vcheck.demo.dev.databinding.LivenessInstructionsFragmentBinding

class LivenessInstructionsFragment : Fragment(R.layout.liveness_instructions_fragment) {

    private var _binding: LivenessInstructionsFragmentBinding? = null

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        _binding = LivenessInstructionsFragmentBinding.bind(view)

        _binding!!.livenessStartButton.setOnClickListener {
            findNavController().navigate(R.id.action_mainFragment_to_faceCheckFragment)
        }
    }
}